src/wallet/rpcwallet.cpp

```diff
@@ -4014,7 +4014,11 @@ UniValue walletcreatefundedpsbt(const JSONRPCRequest& request)
 
     CAmount fee;
     int change_position;
-    CMutableTransaction rawTx = ConstructTransaction(request.params[0], request.params[1], request.params[2], request.params[3]["replaceable"]);
+    bool rbf = pwallet->m_signal_rbf;
+    if (!request.params[3]["replaceable"].isNull()) {
+        rbf = request.params[3]["replaceable"].isTrue();
```

Comment:
Probably ought to throw an error if it's not a boolean here.

Reply:
nm, that's already checked in existing code

Reply:
Added a check.